test/uk/gov/hmrc/gatekeeperemail/connectors/DeveloperConnectorSpec.scala

```diff
@@ -42,7 +43,7 @@ class DeveloperConnectorSpec extends AsyncHmrcSpec
     implicit val hc: HeaderCarrier = HeaderCarrier()
 
     val mockAppConfig = mock[AppConfig]
-    val httpClient    = app.injector.instanceOf[HttpClient]
+    val httpClient    = app.injector.instanceOf[HttpClientV2]
```

Comment:
Instead of injecting the HttpClientV2, could we use the HttpClientV2Support trait instead?
https://github.com/hmrc/http-verbs?tab=readme-ov-file#testing

Reply:
Good idea! I've done that.